metricbeat/module/couchdb/_meta/test/serverstats.json

```diff
@@ -0,0 +1,293 @@
+{
+"couchdb": {
+    "server": {
+      "httpd": {
+        "clients_requesting_changes": {
+          "stddev": 0,
+          "min": 0,
+          "max": 0,
+          "description": "number of clients for continuous _changes",
+          "current": 0,
```

Comment:
Maybe for the purpouse of metrics collected by metricbeat we could collect current value only, aggregated values can be hard to consume correctly.
@ruflin wdyt about these cases?

Reply:
Historically I would say lets only collect the current because of what you mentioned above. At the same time it can sometimes simplify queries having some of the other values around. What I would suggest is that we just start with `current` and have the discussion around adding more fields in a follow up PR. I think creating a dashboard could also be beneficial here as it will make it more obvious if we need the fields for some queries.

Reply:
Let's collect current values only, yes, and without `.current`, so directly in the metric, like in:
```
   "couchdb": {
     "server": {
       "httpd": {
         "clients_requesting_changes": 0,
         "temporary_view_reads": 0,
....
```